new-pipeline/dist.html

```diff
@@ -59,89 +59,92 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                    <div class="caption-div">
```

Comment:
There are a lot of whitespace or indentation changes in this file.
Can you undo those?